### PR TITLE
add ignore to hco-bundle-registry-container

### PR DESCRIPTION
**Pull request by tal-hason**

### CODE REVIEW

**Comment by dominikholler** [comment]
@tal-hason Would removing the relevant lines in https://github.com/openshift/check-payload/blob/main/config.toml to avoid redundancy ?

**Comment by tal-hason** [comment]
> @tal-hason Would removing the relevant lines in https://github.com/openshift/check-payload/blob/main/config.toml to avoid redundancy ?

Not sure, i think it's only catching the versions, I can verify it with CVP slack

File: dist/releases/4.18/config.toml [hunk 1]
```diff
@@ -435,3 +435,17 @@ files = ["/usr/bin/cpb"]
...
+
+[[payload.hco-bundle-registry-container.ignore]]
+error = "ErrGoNotCgoEnabled"
+files = [
+  "/usr/bin/cdi-containerimage-server",
+  "/usr/local/bin/disk-uploader",
+  "/usr/local/bin/kubevirt-tekton-tasks.test",
+  "/usr/bin/ssp-operator.test",
+  "/usr/bin/tekton-tasks-operator.test"
+]
```
Comment (ksimon1) [inline dist/releases/4.18/config.toml:451]: New line is missing at the end of this file

**Comment by codingben** [comment]
> Is there a conversation in slack about these binaries? I'm curious why these are being disabled?

For example, there are `*.test` binaries that are not shipped to the customers, only to QE, and thus no FIPS checks needed. As for others, we just want to ignore this error (I'm pretty sure there is a reason for it in upstream).

**Comment by codingben** [comment]
@tal-hason Hi Tal, thanks! Just to make sure, it's only needed in 4.13, 4.18 and 4.19? I'd suggest to update PR's description to mention it.

**Comment by tal-hason** [comment]
> @tal-hason Hi Tal, thanks! Just to make sure, it's only needed in 4.13, 4.18 and 4.19? I'd suggest to update PR's description to mention it.

From what I understood the 4.18 will effect the 4.17 4.16 4.15 and 4.14
and 4.13 will effect 4.12

am I correct?

**Comment by Axuba** [comment]
Hello, can this be merged, is there anything blocking? We want to make sure we fully pass FIPS checks before the warnings become failures.

**Comment by tal-hason** [comment]
@ksimon1 can you assist with this PR to move to being merged

**Comment by codingben** [comment]
> > @tal-hason Hi Tal, thanks! Just to make sure, it's only needed in 4.13, 4.18 and 4.19? I'd suggest to update PR's description to mention it.
> 
> From what I understood the 4.18 will effect the 4.17 4.16 4.15 and 4.14 and 4.13 will effect 4.12
> 
> am I correct?

I'm not sure about this. I'd ask the maintainers of this repository.

**Comment by tal-hason** [comment]
> > > @tal-hason Hi Tal, thanks! Just to make sure, it's only needed in 4.13, 4.18 and 4.19? I'd suggest to update PR's description to mention it.
> > 
> > 
> > From what I understood the 4.18 will effect the 4.17 4.16 4.15 and 4.14 and 4.13 will effect 4.12
> > am I correct?
> 
> I'm not sure about this. I'd ask the maintainers of this repository.

This is what was told here:
https://redhat-internal.slack.com/archives/C07TC0L7ATB/p1731420233351089?thread_ts=1731412943.825689&cid=C07TC0L7ATB

**Pull request merged**